### PR TITLE
Setup the repository for development

### PR DESCRIPTION
**Pull request by lranjbar**
[ARO-5117](https://issues.redhat.com//browse/ARO-5117)

This PR adds the basic structure of the repository to get started for development.

### CODE REVIEW

File: aro-e2e/common.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+function clean {
+    echo "aro-e2e clean"
+}
+
+function run {
+    echo "Run aro-e2e"
+}
+
+
+## Run function called from command line. ie: `build.sh build`
```
Comment (SudoBrendan) [inline aro-e2e/common.sh:20]: nit: `common.sh build`

**Review by SudoBrendan — APPROVED** [review]
Only nits on what look like copy-paste errors. Looks like a good structure to me :)

File: classic/conftest.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+function cluster-conftest {
+    echo "Run OCP conformance tests"
+}
+
+## Run function called from command line. ie: `config.sh workflow-classic-config`
```
Comment (SudoBrendan) [inline classic/conftest.sh:7]: nit: `conftest.sh cluster-conftest`

**Review by jhoreman — APPROVED** [review]
Great first commit. Thanks for this.

**Review by tsatam — APPROVED** [review]
I like what I see here! I think this will be a good foundation for to start this new repository and process.

**Review by kimorris27 — APPROVED** [review]

**Review by hlipsig — APPROVED** [review]

**Pull request merged**